packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js

```diff
@@ -84,7 +114,7 @@ function ActionDropdown(props) {
 			className={classNames(theme['tc-dropdown-button'], 'tc-dropdown-button')}
 			{...rest}
 		>
-			{items.length ? items.map(getMenuItem) : <MenuItem disabled>No options</MenuItem>}
+			{chooseMenuItemRendering(getComponent, items, components)}
```

Comment:
ok to remove the no options but why do you want to chose between both.
I would use
```javascript
const injected = Inject.all(getComponent, components);
...
{injected('pre-items')}
{items.length && items.map(getMenuItem)}
{injected('post-items')}
```